### PR TITLE
AudibleInc.AudibleSync version 1.8.8

### PR DESCRIPTION
**Pull request by BattlemasterLoL**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15604)

### CODE REVIEW

File: manifests/a/AudibleInc/AudibleSync/1.8.8/AudibleInc.AudibleSync.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: AudibleInc.AudibleSync
+PackageVersion: 1.8.8
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://windows-lite-update.s3.amazonaws.com/windows/v1.8.8/AudibleSync+Setup+1.8.8.exe?pf_rd_p=414475fc-cc54-4d14-838c-b7c61b8f72c7
+  InstallerSha256: BA1228FB9E37D142700EF63A6CBB838B97E3745057E39B4D6B9086364D3167E9
```
Comment (jedieaston) [inline manifests/a/AudibleInc/AudibleSync/1.8.8/AudibleInc.AudibleSync.installer.yaml:9]:
```suggestion
  InstallerSha256: BA1228FB9E37D142700EF63A6CBB838B97E3745057E39B4D6B9086364D3167E9
  InstallerSwitches: 
      Silent: " "
      SilentWithProgress: " "
```
To stop the warning for `winget validate`.

**Review by jedieaston — CHANGES_REQUESTED** [review]

File: manifests/a/AudibleInc/AudibleSync/1.8.8/AudibleInc.AudibleSync.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate
+
+PackageIdentifier: AudibleInc.AudibleSync
```
Comment (ItzLevvie) [inline manifests/a/AudibleInc/AudibleSync/1.8.8/AudibleInc.AudibleSync.installer.yaml:3]:
```suggestion
PackageIdentifier: Audible.AudibleSync
```

File: manifests/a/AudibleInc/AudibleSync/1.8.8/AudibleInc.AudibleSync.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: AudibleInc.AudibleSync
```
Comment (ItzLevvie) [inline manifests/a/AudibleInc/AudibleSync/1.8.8/AudibleInc.AudibleSync.yaml:3]:
```suggestion
PackageIdentifier: Audible.AudibleSync
```

File: manifests/a/AudibleInc/AudibleSync/1.8.8/AudibleInc.AudibleSync.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: AudibleInc.AudibleSync
```
Comment (ItzLevvie) [inline manifests/a/AudibleInc/AudibleSync/1.8.8/AudibleInc.AudibleSync.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: Audible.AudibleSync
```

File: manifests/a/AudibleInc/AudibleSync/1.8.8/AudibleInc.AudibleSync.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: AudibleInc.AudibleSync
+PackageVersion: 1.8.8
+PackageLocale: en-US
+Publisher: Audible Inc
+PackageName: AudibleSync
+License: Copyright © 2021 Audible Inc
+ShortDescription: AudibleSync
```
Comment (ItzLevvie) [inline manifests/a/AudibleInc/AudibleSync/1.8.8/AudibleInc.AudibleSync.locale.en-US.yaml:9]:
```suggestion
ShortDescription: The AudibleSync app is a Windows 10 app for your computer that allows for quick and easy downloading of your titles.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You also have to use the GitHub client to change the folder name and the file names.

**Pull request closed without merging**